pkg/providers/vsphere/virtualmachine/backup.go

```diff
@@ -240,7 +242,7 @@ func canPerformBackup(opts BackupVirtualMachineOptions, backupVersionEcVal, back
 
 	storedVersion, err := strconv.ParseInt(backupVersionEcVal, 10, 64)
 	if err != nil {
-		opts.VMCtx.Logger.Error(err, fmt.Sprintf("failed to parse backup version extraConfig key to int64 : %v", backupVersionEcVal))
+		opts.VMCtx.Logger.Error(err, fmt.Sprintf("failed to parse backup version extraConfig key to int64: %v", backupVersionEcVal))
```

Comment:
```suggestion
		opts.VMCtx.Logger.Error(err, "failed to parse backup version ExtraConfig value to int64", "backupVersionECVal", backupVersionEcVal)
```

So it is key-able when searching.